### PR TITLE
founding chapter tweaks

### PR DESCRIPTION
**Pull request by KRdaMystic**
Company, role, personnel, and flagship names added or changed

### CODE REVIEW

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 1]
```diff
@@ -62,10 +63,17 @@ function scr_chapter_new(argument0) {
 	    disposition[7]=0;// Reserved
 	    chapter_master_name="Azreal";chapter_master_specialty=2;
 	    chapter_master_melee=4;chapter_master_ranged=4;
-	    company_title[1]="Deathwing";
-	    company_title[2]="Ravenwing";
+		
+	    company_title[1]="Deathwing";company_title[2]="Ravenwing";company_title[3]="The Unmerciful"
+		company_title[4]="The Feared"company_title[5]="The Unrelenting"company_title[6]="The Resolute"
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:68]:
```suggestion
		company_title[4]="The Feared";
		company_title[5]="The Unrelenting";
		company_title[6]="The Resolute";
```

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 2]
```diff
@@ -62,10 +63,17 @@ function scr_chapter_new(argument0) {
 	    disposition[7]=0;// Reserved
 	    chapter_master_name="Azreal";chapter_master_specialty=2;
 	    chapter_master_melee=4;chapter_master_ranged=4;
-	    company_title[1]="Deathwing";
-	    company_title[2]="Ravenwing";
+		
+	    company_title[1]="Deathwing";company_title[2]="Ravenwing";company_title[3]="The Unmerciful"
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:67]:
```suggestion
	    company_title[1]="Deathwing";
	    company_title[2]="Ravenwing";
	    company_title[3]="The Unmerciful";
```

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 3]
```diff
@@ -62,10 +63,17 @@ function scr_chapter_new(argument0) {
...
+		company_title[4]="The Feared"company_title[5]="The Unrelenting"company_title[6]="The Resolute"
+		company_title[7]="The Unbowed"company_title[8]="The Wrathful"company_title[9]="The Remorseless"
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:69]:
```suggestion
		company_title[7]="The Unbowed";
		company_title[8]="The Wrathful";
		company_title[9]="The Remorseless";
```

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 4]
```diff
@@ -62,10 +63,17 @@ function scr_chapter_new(argument0) {
...
+		company_title[10]="The Redeemed"
+		
+		i=99;repeat(3){i+=1;
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:72]:
```suggestion
		for(i=100;i<=102;i++){
```

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 5]
```diff
@@ -211,6 +211,16 @@ function scr_initialize_custom() {
 			strike_cruisers=8;
 	        gladius=7;
 	        hunters=3;
+			
+			if (global.chapter_name="Salamanders")
```
Comment (OH296) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:215]: probably use a switch statement here

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 6]
```diff
@@ -226,8 +236,13 @@ function scr_initialize_custom() {
 				   strike_cruisers-=4
 			   }
 		
-	        if (global.chapter_name="Dark Angels") then battle_barges+=1;
-	    }
+	        if (global.chapter_name="Dark Angels") 
+			{
+			flagship_name="Invincible Reason";
+			battle_barges+=1;
```
Comment (OH296) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:242]:
```suggestion
			      flagship_name="Invincible Reason";
			      battle_barges++;
```

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 7]
```diff
@@ -268,13 +272,14 @@ if (argument0="White Scars"){founding="N/A";points=150;
...
     
-	    company_title[1]="Clan Avernii";company_title[2]="Clan Garrsak";company_title[3]="Clan Felg";
-	    company_title[4]="Clan Garrsak";company_title[5]="Clan Kaargul";company_title[6]="Clan Morragul";
-	    company_title[7]="Clan Sorrgol";company_title[8]="Clan Borrgar";company_title[9]="Clan Ungavarr";
+	    company_title[1]="Clan Avernii";company_title[2]="Clan Garrsak";company_title[3]="Clan Raukaan";
+	    company_title[4]="Clan Kaargul";company_title[5]="Clan Haarmek";company_title[6]="Clan Sorrgol";
+	    company_title[7]="Clan Borrgos";company_title[8]="Clan Morlaag";company_title[9]="Clan Vurgaan";
 	    company_title[10]="Clan Dorrvok";
     
 	    i=99;repeat(3){i+=1;
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:280]:
```suggestion
	   for(i=100;i<=102;i++){
```

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 8]
```diff
@@ -62,10 +63,17 @@ function scr_chapter_new(argument0) {
...
-	    company_title[1]="Deathwing";
-	    company_title[2]="Ravenwing";
+		
+	    company_title[1]="Deathwing";company_title[2]="Ravenwing";company_title[3]="The Unmerciful"
+		company_title[4]="The Feared"company_title[5]="The Unrelenting"company_title[6]="The Resolute"
+		company_title[7]="The Unbowed"company_title[8]="The Wrathful"company_title[9]="The Remorseless"
+		company_title[10]="The Redeemed"
+		
+		i=99;repeat(3){i+=1;
+		role[i,5]="Master";wep1[i,5]="Power Sword";
```
Comment (OH296) [inline scripts/scr_chapter_new/scr_chapter_new.gml:73]:
```suggestion
		role[i,5]="Master";
		wep1[i,5]="Power Sword";
```

**Review by OH296 — COMMENTED** [review]
basically all fine just made some suggestions but essentially this file needs a refactor

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 9]
```diff
@@ -50,7 +50,8 @@ function scr_chapter_new(argument0) {
 	    color_to_main="Dark Green";color_to_secondary="Dark Green";color_to_trim="Silver";
 	    color_to_pauldron="Dark Green";color_to_pauldron2="Dark Green";color_to_lens="Dark Red";
 	    color_to_weapon="Black";col_special=0;trim=0;
-	    hchaplain="Sapphon";clibrarian="Ezekial";
+	    hchaplain="Sapphon";clibrarian="Ezekial";fmaster="Sepharon";hapothecary="Razaek";
+		admiral="Korahael";recruiter="Ranaeus"
```
Comment (GalacticChimp) [inline scripts/scr_chapter_new/scr_chapter_new.gml:54]: missing `;` after `"Ranaeus"`

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
please add missing `;`, apart from that PR looks fine

**Comment by ZAB909** [comment]
Hey sorry for not informing but I'm currently adding a new chapter and this would overwrite. I'll try to get it done asap.

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 5 above)
Comment (KRdaMystic) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:234]: I'm still very new to coding so I'm not sure what a "switch statement" is but if you show me how I can implement it in the future.

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 10]
```diff
@@ -62,10 +63,24 @@ function scr_chapter_new(argument0) {
...
 	    company_title[1]="Deathwing";
 	    company_title[2]="Ravenwing";
+	    company_title[3]="The Unmerciful";
+		company_title[4]="The Feared";
+		company_title[5]="The Unrelenting";
+		company_title[6]="The Resolute";
+		company_title[7]="The Unbowed";
+		company_title[8]="The Wrathful";
+		company_title[9]="The Remorseless";
+		company_title[10]="The Redeemed"
```
Comment (GalacticChimp) [inline scripts/scr_chapter_new/scr_chapter_new.gml:76]: `;`

File: scripts/scr_chapter_new/scr_chapter_new.gml [hunk 11]
```diff
@@ -289,7 +301,7 @@ if (argument0="White Scars"){founding="N/A";points=150;
 	    color_to_main="Green";color_to_secondary="Green";color_to_trim="Copper";
 	    color_to_pauldron="Black";color_to_pauldron2="Black";color_to_lens="Dark Red";
 	    color_to_weapon="Black";col_special=0;trim=0;
-	    fmaster="Vulkan He'stan";
+	    fmaster="Argos";clibrarian="Velcona";hapothecary="Harath Shen";recruiter="Sol Ba'ken";admiral="Dac'tyr";hchaplain="Elysius"
```
Comment (GalacticChimp) [inline scripts/scr_chapter_new/scr_chapter_new.gml:304]: `;`

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 5 above)
Comment (GalacticChimp) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:234]: https://manual.yoyogames.com/GameMaker_Language/GML_Overview/Language_Features/switch.htm

but if you don't feel comfortable with GML yet, then just leave the ifs, it's fine here

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 12]
```diff
@@ -230,6 +230,16 @@ function scr_initialize_custom() {
...
 	        hunters=3;
+			
+			if (global.chapter_name="Salamanders")
+				{
+					flagship_name="Flamewrought"
+				}
+			
+			if (global.chapter_name="Ultramarines")
+				{
+					flagship_name="Laurels of Victory"
```
Comment (GalacticChimp) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:241]: `;`

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
only 3 missing `;`, after this is fixed I'll merge this one

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**